packages/api-client/src/components/ApiClient/Request/Request.vue

```diff
@@ -7,7 +7,13 @@ import RequestHeaders from './RequestHeaders.vue'
 import RequestQuery from './RequestQuery.vue'
 import RequestVariables from './RequestVariables.vue'
 
-const { activeRequest, readOnly } = useRequestStore()
+const { activeRequest } = useRequestStore()
```

Comment:
we should keep `readOnly`

and have a config object at root, then we can do

```
        <ApiClient
          :proxyUrl="proxyUrl"
          :readOnly="readOnly"
          theme="none"
          @escapeKeyPress="hideApiClient" />
```

then inside of 
```
// Whether the request is in read mode or edit mode
const readOnly = ref(true)

export const useRequestStore = () => ({
  authState,
  readOnly,
  activeRequest,
  activeResponse,
  requestHistory,
  requestHistoryOrder,
  activeRequestId,
  setActiveResponse,
  resetActiveResponse,
  addRequestToHistory,
  setActiveRequest,
})

```

we can add a `setReadOnly` :)

Reply:
Hm … I removed the `readOnly` just from the `Request.vue` file, so the operation name isn’t editable - because it doesn’t have an effect anyway.

I kept the `readOnly` in the store and we still use it to determine whether the address is editable or not.

Does that make sense?

With your proposed change the operation name would become an input again (changes wouldn’t have an effect).